swatch-producer-aws/src/main/java/com/redhat/swatch/files/TagProfile.java

```diff
@@ -39,7 +40,8 @@ public class TagProfile {
   private final Map<TagMetricLookupKey, Double> billingFactorLookup;
 
   public TagProfile() {
-    Representer representer = new Representer();
+    DumperOptions dumperOptions = new DumperOptions();
+    Representer representer = new Representer(dumperOptions);
```

Comment:
I create the Yaml object slightly differently in [SubscriptionRegistry.java](https://github.com/RedHatInsights/rhsm-subscriptions/blob/410da302a2a13a8990277312be90916867e1deb6/swatch-product-configuration/src/main/java/com/redhat/swatch/configuration/registry/SubscriptionRegistry.java#L64-L67).  Do you know if one way is preferred over the other?  No need to change this because `TagProfile` is on its way out, but just want to know if I should update SubscriptionRegistry to match this instead.

Reply:
Nah, seems fine to do either. Probably better to do the more targeted way you've done in SubscriptionRegistry :smile: